src/pretix/base/models/items.py

```diff
@@ -110,16 +113,116 @@ class ItemCategory(LoggedModel):
                     'only be bought in combination with a product that has this category configured as a possible '
                     'source for add-ons.')
     )
+    CROSS_SELLING_MODES = (
```

Comment:
I'm a bit hesitant to change that, as it'd need changes in many places, we'd need to keep API compatibility anyway, and I don't see it as a much better representation (it would match the UI, but on the other hand, these are two quite different properties of a category - is_addon is something that's really enforced in the cart logic, whereas the cross-selling mode is just presentational).

I don't have a strong opinion on it, so if you do think it's a lot better, I can change it, but it's quite some work and I'm not sure whether it's worth the effort.